server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -235,7 +280,26 @@ export class TenantManager {
     /**
      * Retrieves the secret for the given tenant
      */
-    public async getTenantKeys(tenantId: string, includeDisabledTenant = false): Promise<ITenantKeys> {
+    public async getTenantKeys(tenantId: string, includeDisabledTenant = false, disableCache=false): Promise<ITenantKeys> {
+        const lumberProperties = { [BaseTelemetryProperties.tenantId]: tenantId, includeDisabledTenant };
+        const fetchTenantKeyMetric = Lumberjack.newLumberMetric(LumberEventName.RiddlerFetchTenantKey);
+        let uncaughtException;
+        let retrievedFromCache;
+
+
+        try {
+            retrievedFromCache = false;
```

Comment:
You can also optionally avoid this line by initializing `retrievedFromCache` in line 287 directly, like:
`let retrievedFromCache = false;`